src/Compilers/CSharp/Portable/FlowAnalysis/LocalDataFlowPass.cs

```diff
@@ -162,16 +162,10 @@ private int DescendThroughTupleRestFields(ref Symbol symbol, int containingSlot,
                         return -1;
                     }
 
-                    if (forceContainingSlotsToExist)
+                    containingSlot = GetOrCreateSlot(restField, containingSlot, createIfMissing);
```

Comment:
>containingSlot = GetOrCreateSlot(restField, containingSlot, createIfMissing); [](start = 20, length = 77)

Is propagating ```createIfMissing``` has any real effect here? It looks like ```restField``` is coming from an underlying named type. Therefore, it cannot be a TupleFieldSymbol. Therefore, the ```DescendThroughTupleRestFields``` call in the ```GetOrCreateSlot``` above will be a NoOp regardless of the value passed in ```createIfMissing```.
If these observations are correct, then it means that now ```createIfMissing``` has no effect on the "result" of this function, but it looks like the ```forceContainingSlotsToExist``` parameter used to have an effect.